addons/website/static/src/snippets/s_dynamic_snippet/000.js

```diff
@@ -190,11 +190,11 @@ const DynamicSnippet = publicWidget.Widget.extend({
      */
     _setupCallToAction: function (enable) {
         if (enable === true) {
-            this.$el.on('click.s_dynamic_snippet', '*[data-url]', function (ev) {
-                if (this.editableMode !== true) {
+            if (this.editableMode !== true) {
+                this.$el.on('click.s_dynamic_snippet', '*[data-url]', function (ev) {
                     window.location = $(ev.currentTarget).attr('data-url');
-                }
-            }.bind(this));
+                }.bind(this));
+            }
```

Comment:
Any reason this whole `_setupCallToAction` method exist? Can't you use the `read_events` property? (and before this fix, the standard `events` property and move this code in a handler method?

Reply:
👍